app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: %i[edit update destroy]
+
+  def new
+    @comment = Comment.new
```

Comment:
PostへのCommentになるので、URLは以下のようになるはず。
`/posts/1/comments/new` 

そうすると、before_action で `set_post` を用意しておいて、`@post.comments.build` を使うようにする。
```suggestion
    @comment = @post.comments.build
```